frontend/pages/docs/users/using-a-model/downloading-files.mdx

```diff
@@ -0,0 +1,23 @@
+import DocsWrapper from 'src/docs/DocsWrapper'
+import Image from 'next/legacy/image'
+import Box from '@mui/material/Box'
+
+import releases from 'public/docs/bailo_releases.png'
+import release from 'public/docs/bailo_release.png'
+
+## How to download files
+
+Files are attached to model releases. You can download files from a release in one of two ways - by either finding the
+release in the `Releases` tab and clicking the file download link:
+
+<Box sx={{ maxWidth: '75%', margin: 'auto' }}>
+  <Image src={releases} />
+</Box>
+
+or alternatively, visiting the page for a specific release and clicking the file download link:
```

Comment:
I'd probably want a line here saying that you either need to be a model developer or have an approved access request.  That sentence should also likely link to 'requesting access' for more information.  Otherwise, I'd just read this page, try it out and be sad it doesn't work!